<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>正在分割</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>正在分割，请等待……</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="run_log">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>60</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>60</number>
       </property>
       <item>
        <widget class="QPushButton" name="mute_run_btn">
         <property name="text">
          <string>在后台运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="interrupt_btn">
         <property name="text">
          <string>中断</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
